tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/AspNetCore2IastTests.cs

```diff
@@ -88,6 +88,7 @@ public class AspNetCore2IastTestsFullSamplingEnabled : AspNetCore2IastTestsFullS
     public AspNetCore2IastTestsFullSamplingEnabled(AspNetCoreTestFixture fixture, ITestOutputHelper outputHelper)
         : base(fixture, outputHelper, enableIast: true, testName: "AspNetCore2IastTestsEnabled", isIastDeduplicationEnabled: false, vulnerabilitiesPerRequest: 200)
     {
+        SetEnvironmentVariable("ASPNETCORE_ENVIRONMENT", "Development");
```

Comment:
Interesting 🤔 Totally understand why you need this, but also, you may end up with issues later for APIs that need it _not_ to be this.

Also, it might be nice to have a test where you don't send this, to confirm there _is_ no stack-leak vulnerability

Reply:
I guess that if in the future we have apis in our tests that don't need that, we could have two different test classes, one with each value. I did not add tests without the developer mode for CI performance, but I can add one just to make sure.